lib/country-iso-3166-1-alpha-3-codes.ts

```diff
@@ -193,3 +195,7 @@ export const countryNameToIso31661Alpha3CodeMap: Record<string, string> = {
   "St Lucia": "LCA",
   "Saint Lucia": "LCA",
 };
+
+export const iso31661Alpha3CodeToCountryNameMap: Record<string, string> = typedObjectFromEntries(
+  typedObjectEntries(countryNameToIso31661Alpha3CodeMap).map(([key, value]) => [value, key])
```

Comment:
If you're curious what name it uses if there's multiple entries with the same alpha 3 country code, it uses the last one in the list. Pretty minor detail, I want to move all of this stuff to the backend anyways.